cmd/preset.go

```diff
@@ -84,30 +88,30 @@ func (i *KoolInit) Execute(args []string) (err error) {
 	return
 }
 
-// NewInitCommand initializes new kool init command
```

Comment:
Above, at line 82:

![image](https://user-images.githubusercontent.com/1116377/96632212-c36a3a00-12ed-11eb-8a30-8b3299020de2.png)


I may have overlooked how you dealt with this, but won't we start creating a `preset_language` file as well now? I guess we DO NOT want that, so we need to have some kind of check to avoid doing it?